crates/edr_opt/tests/full_block.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg(feature = "test-remote")]
+
+use edr_evm::impl_full_block_tests;
+use edr_opt::OptimismChainSpec;
+use edr_test_utils::env::get_alchemy_url;
+
+impl_full_block_tests! {
+    // This test often failes with an internal Alchemy error:
+    // > execution aborted (timeout = 10s)
+    //
+    // Potentially the block is too old?
+    // mainnet_pre_bedrock => OptimismChainSpec {
+    //     block_number: 98_235_064,
+    //     chain_id: 10,
+    //     url: get_alchemy_url().replace("eth-", "opt-"),
+    // },
```

Comment:
Should this be removed?

Reply:
Addressed by https://github.com/NomicFoundation/edr/pull/534/commits/cbdb02b85bf632a5d49e379fa5407a8dd39e55e3